app/utils/Constants.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+object Constants {
+  final val GB = "GB"
+  final val ES = "ES"
+  final val UNITED_KINGDOM = "United Kingdom"
+  final val SPAIN = "Spain"
+}
```

Comment:
Think there's an existing `Constant` object that already has `GB`. I prefer the name `Constants` though so you can delete the other one and point the existing usages to this one instead

Reply:
Updated